test/xpu_api/numerics/c.math/abs1.pass.cpp

```diff
@@ -69,10 +69,8 @@ ONEDPL_TEST_NUM_MAIN
     test_abs<std::int32_t, correct_size_int<std::int32_t>::type>();
     test_abs<std::int64_t, correct_size_int<std::int64_t>::type>();
 
-    if constexpr (HasLongDoubleSupportInCompiletime::value)
-        test_abs<long double, long double>();
-    if constexpr (HasDoubleSupportInRuntime::value)
-        test_abs<double, double>();
+    TestUtils::invoke_test_if<>(HasLongDoubleSupportInCompiletime(), []() { test_abs<long double, long double>(); });
+    TestUtils::invoke_test_if<>(HasDoubleSupportInRuntime(), []() { test_abs<double, double>(); });
```

Comment:
You do not have to specify empty angle brackets when calling a function, do you?

Reply:
yes. Fixed.